baybe/utils/botorch_wrapper.py

```diff
@@ -1,27 +1,49 @@
-"""A wrapper class for synthetic BoTorch test functions."""
+"""Wrapper functionality synthetic BoTorch test functions."""
 
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Iterable
+
+import pandas as pd
+import torch
 from botorch.test_functions import SyntheticTestFunction
-from torch import Tensor
+
+if TYPE_CHECKING:
+    pass
 
 
-def botorch_function_wrapper(test_function: SyntheticTestFunction):
-    """Turn a BoTorch test function into a format accepted by lookup in simulations.
+def add_dataframe_layer(
+    test_function: SyntheticTestFunction,
+    output_columns: Iterable[str],
+):
+    """Wrap a BoTorch test function into a format suitable for lookups in simulations.
 
-    See :mod:`baybe.simulation` for details.
+    The wrapper layer removes the dataframe indices, passes the raw numeric values
+    as tensor to the test function, and turns the response into a dataframe with the
+    given column names. The See :mod:`baybe.simulation.lookup` for details on the
```

Comment:
Typo/weird sentence

Reply:
What exactly? 😃

Reply:
`The See :mod:`baybe.simulation.lookup` for details on the lookup is used.`